packages/adp-tooling/templates/rta/common/grid-tree-custom-column.xml

```diff
@@ -0,0 +1,19 @@
+<!-- Use stable and unique id's!-->
+<core:FragmentDefinition xmlns="sap.m" xmlns:table="sap.ui.table" xmlns:smartfield="sap.ui.comp.smartfield"	xmlns:core="sap.ui.core">
```

Comment:
smartfiled namespace can be removed as it is not used

Reply:
True. Will remove it in the next PR.